code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -10,16 +10,21 @@
 	var/rigged = 0
 	var/sound_effect_open = 'sound/machines/click.ogg'
 	var/sound_effect_close = 'sound/machines/click.ogg'
-
+	var/obj/item/weapon/paper/manifest/manifest
+	
 /obj/structure/closet/crate/New()
 	..()
+	overlays.Cut()

```

Comment:
Should I also move the secure's crate addition of overlay to the update_icon()? or should I instead add the update_icon() to the new() before the addition of the overlay?